shell/platform/fuchsia/flutter/vsync_waiter.h

```diff
@@ -14,40 +14,64 @@
 #include "flutter/shell/common/vsync_waiter.h"
 #include "flutter_runner_product_configuration.h"
 
+#include "session_connection.h"
+
 namespace flutter_runner {
 
+// (frame_start_time, frame_target_time)
+using FrameTargetTimes = std::pair<fml::TimePoint, fml::TimePoint>;
+// (latch point time, vsync time)
+using LatchPointVsyncPair = std::pair<fml::TimePoint, fml::TimePoint>;
+
 class VsyncWaiter final : public flutter::VsyncWaiter {
  public:
   static constexpr zx_signals_t SessionPresentSignal = ZX_EVENT_SIGNALED;
-
-  VsyncWaiter(std::string debug_label,
-              zx_handle_t session_present_handle,
+  VsyncWaiter(SessionConnection* session_connection,
```

Comment:
Nit: please move constructor below static methods (google style guide dictates this)

Reply:
Done